app/controllers/api/ims_exports_controller.rb

```diff
@@ -10,23 +10,24 @@ def show
   end
 
   def status
+    export = ImsExport.find_by(token: params[:id])
     respond_to do |format|
-      format.json { render json: { status: "completed" } }
+      format.json do
+        render json: { status: export.status, message: export.message }
+      end
     end
   end
 
   def create
-    lti_launches = LtiLaunch.where(context_id: params[:context_id])
-    payload = {
-      application_instance_id: current_application_instance.id,
-      context_id: params[:context_id],
-      lti_launch_tokens: lti_launches.pluck(:token),
-    }
     export = ImsExport.create!(
       tool_consumer_instance_guid: params[:tool_consumer_instance_guid],
       context_id: params[:context_id],
       custom_canvas_course_id: params[:custom_canvas_course_id],
-      payload: payload,
+    )
+    ImsExportJob.perform_later(
```

Comment:
Do we require active job now? (that's fine just checking)

Reply:
Nope. It is using ActiveJob is all.